biom/table.py

```diff
@@ -635,28 +637,34 @@ def add_metadata(self, md, axis='sample'):
 
         Parameters
         ----------
-        md : dict of dict
+        md : dict of dict or pd.DataFrame
```

Comment:
Yup.  I'm doing an explicit casting to pandas on [line 650](https://github.com/mortonjt/biom-format/blob/0997ce172415d47a7e80cfa78f2893d07a57f019/biom/table.py#L650
) and [line 654](https://github.com/mortonjt/biom-format/blob/0997ce172415d47a7e80cfa78f2893d07a57f019/biom/table.py#L654)